logging/logging-getting-started.adoc

```diff
@@ -0,0 +1,58 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="logging-getting-started"]
+= Getting started with {logging}
+include::_attributes/common-attributes.adoc[]
+:context: <logging-getting-started>
+
+toc::[]
+
+
+////
+A note for reviewers: This should have been a reference module, but it would be a wasted opportunity to not link to documentation for each feature, and we can't do that in a module because we can't xref in a module. Can't easily nest assemblies either, so instead of a new page just for feature reference, this is probably the first piece of a getting started assembly.
+////
+
+
+[id="logging-features-by-version_{context}"]
+== {logging-uc} features by version
+
+Each version of {logging} introduces new features. Features added in earlier versions are generally maintained in subsequent releases unless otherwise specified. This summary of major features has been organized by initial release version.
+
+//Dep notices, or a statement about development direction?
+
+=== {logging-uc} 5.8
+
+.Loki Operator
+* Zone Aware Data Replication
+* Restart and Reliability Hardening
+* Dashboards
+* Fine Grained Logs Access
+* Developer Logs Based Alerts
+* Support for 1x.extra.small size
+
+.Cluster Logging Operator
+* Flow control/rate limiting
+* Audit Log Filtering
+* Cluster Logging HTTP Server
+
+.Vector collector
+* Compliance with TLS Security profile configuration for server endpoints
+* Support for TLS Security profiles that matches the output
+* Multi ClusterLogForwarder
```

Comment:
Please use the attributes for the Operator names. If you need to update the {clo} one that's fine, but I want us to stay in the habit of using the attributes for consistency.

Reply:
All of these points should be sentence case as well

Reply:
Oh this isn't remotely peer-review ready, I was just compiling the initial list & figuring out presentation form. Most of those are copy-pasted from the enablement slides.  :)